functions.php

```diff
@@ -94,9 +94,8 @@ function extract_regions_from_db(PDO $db): array {
  * @param PDO $db , the db you want to add the values to
  */
 function add_new_item_to_db(array $postArray,PDO $db) {
-    settype($postArray['brand'], "integer");
-    settype($postArray['cost'], "float");
-    settype($postArray['region'], "integer");
-    $query = $db->prepare('INSERT INTO `wine` (`variety`, `tones`, `brand_id`, `cost`, `region_of_origin`) VALUES (?,?,?,?,?);');
-    $query->execute([$postArray['variety'], $postArray['tones'], $postArray['brand'], $postArray['cost'], $postArray['region'] ]);
+    $query = $db->prepare('INSERT INTO `wine` (`variety`, `tones`, `brand_id`, `cost`, `region_of_origin`) 
+            VALUES (:variety, :tones, :brandId , :cost, :regionId);');
+    $query->execute(['variety' => $postArray['variety'], 'tones' => $postArray['tones'],
```

Comment:
here you are taking values out of an array to put them back into an identical array